docs/partial_source/design/building_blocks.rst

```diff
@@ -406,8 +409,9 @@ Now, let's take another example, this time setting the desired backend as tensor
    :align: center
    :width: 75%
 
-In both the examples, we observe that the same Ivy code, is converted into different computation graphs, based on the set framework. Similarly, we've also observed that using a mix of
-Ivy and the desired framework code, also creates the same computation graph as using pure Ivy code. Therefore, all Ivy code is converted into the set backend specific computation graph.
+Additionally, in the above example we also observe that Ivy code and native backend code is compiled to the desired backend code for any framework, not just torch.
```

Comment:
replace all of this with:

The example above further emphasizes that the graph compiler creates a computation graph consisting of _backend_ functions, not _Ivy_ functions. Specifically, the same Ivy code compiles to different graphs depending on the selected backend. However, when compiling native framework code, we are only able to compile a graph for that same framework. For example, we cannot take `torch` code and _compile_ this into `tensorflow` code. However, we can _transpile_ `torch` code into `tensorflow` code (see :ref:`Ivy as a Transpiler` for more details).

Reply:
This is definitely what I was trying to convey too, but much clearer and succinct